<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DFU App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="dfuButton">
      <property name="text">
       <string>Enter DFU Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="checkButton">
      <property name="text">
       <string>Check Device Connection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="alertButton">
      <property name="text">
       <string>App is Working</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
